src/components/NcRichContenteditable/NcRichContenteditable.vue

```diff
@@ -55,84 +57,48 @@ This component displays contenteditable div with automated `@` [at] autocompleti
 		<p class="pre-line">{{ message }}</p>
 
 		<h5>Output - in NcRichText with markdown support</h5>
-		<NcRichText :text="message" use-markdown />
+		<NcRichText :text="text" :arguments="userMentions" autolink use-markdown />
 	</div>
 </template>
 <script>
-	import NcRichText from "../NcRichText/NcRichText.vue";
+	import { parseUserMentions, userData, userMentions } from "./docsHelpers.js";
 
 export default {
-	components: {NcRichText},
-
 	data() {
 		return {
 			message: '**Lorem ipsum** dolor sit amet.',
 
 			// You need to provide this for the inline
 			// mention to understand what to display or not.
-			userData: {
-				Test01: {
-					icon: 'icon-user',
-					id: 'Test01',
-					title: 'Test01',
-					source: 'users',
-					primary: true,
-				},
-				Test02: {
-					icon: 'icon-user',
-					id: 'Test02',
-					title: 'Test02',
-					source: 'users',
-					status: {
-						clearAt: null,
-						icon: '🎡',
-						message: 'Visiting London',
-						status: 'away',
-					},
-					subline: 'Visiting London',
-				},
-				'Test@User': {
-					icon: 'icon-user',
-					id: 'Test@User',
-					title: 'Test 03',
-					source: 'users',
-					status: {
-						clearAt: null,
-						icon: '🎡',
-						message: 'Having space in my name',
-						status: 'online',
-					},
-					subline: 'Visiting London',
-				},
-				'Test Offline': {
-					icon: 'icon-user',
-					id: 'Test Offline',
-					title: 'Test Offline',
-					source: 'users',
-					status: {
-						clearAt: null,
-						icon: null,
-						message: null,
-						status: 'offline',
-					},
-					subline: null,
-				},
-				'Test DND': {
-					icon: 'icon-user',
-					id: 'Test DND',
-					title: 'Test DND',
-					source: 'users',
-					status: {
-						clearAt: null,
-						icon: null,
-						message: 'Out sick',
-						status: 'dnd',
-					},
-					subline: 'Out sick',
-				},
-			}
+			userData,
+			// Test02: {
+			//   icon: 'icon-user',
+			//       id: 'Test02',
+			//       title: 'Test02',
+			//       source: 'users',
+			//       status: {
+			//     clearAt: null,
+			//         icon: '🎡',
+			//         message: 'Visiting London',
+			//         status: 'away',
+			//   },
+			//   subline: 'Visiting London',
+			// },
```

Comment:
Supposed to be removed?
```suggestion
```

Reply:
`userData` is referenced in the JSDOC description of method below:

https://github.com/nextcloud-libraries/nextcloud-vue/blob/c338863ba242294e68e442980d2974c57a315c38/src/components/NcRichContenteditable/NcRichContenteditable.vue#L141

Now as it's moved to the separate file, and is no longer visible in the styleguide code overview, I've decided to keep an example in docs for developers